package-lock.json

```diff
@@ -22850,7 +22854,7 @@
 			"dependencies": {
 				"clone-deep": {
 					"version": "0.2.4",
-					"resolved": "http://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
+					"resolved": "https://registry.npmjs.org/clone-deep/-/clone-deep-0.2.4.tgz",
```

Comment:
This value has a bad tendency to flip back and forth in local environments. It's not strictly part of your changes here, but I think I've seen the same locally, so it might be good to include this anyways, just to avoid future issues.

Reply:
Rights, it always shows up for me locally as https. I even updated it in `master` a few days ago:
https://github.com/WordPress/gutenberg/commit/fe258a64eed124597e37c5f0ed588e395241bf25

There might be something going on with operating systems or versions of npm client.